examples/benchmark/README.md

```diff
@@ -0,0 +1,91 @@
+# Performance evaluation
+
+With the help of this guide you are able to take performance measurments in your setup using STUNner. Both running STUNner locally (outside of Kubernetes) and running STUNner in Kubernetes can be evaluated.
+
+Compare the locally measured results to the result measured in Kubernetes and figure out the overhead cost. The extra cost of your cluster's networking may surprise you in terms of extra delay or more packet drops using the same bandwidth.
+
+### Prerequisites
+
+The tutorial assumes a fresh STUNner installation; see the [STUNner installation and configuration
+guide](/doc/INSTALL.md). Create a namespace called `stunner` if there is none. 
+You must have [`iperfv2`](https://iperf.fr), [`jq`](https://stedolan.github.io/jq/) and most importantly [Go](https://go.dev/doc/install) installed locally to run this tutorial.
+
+## Installation
+
+Install it in case you would like to benchmark your Kubernetes setup. If you want to benchmark locally skip this step.
+
+Install the STUNner gateway operator and STUNner ([more info](https://github.com/l7mp/stunner-helm)):
```

Comment:
Isn't this step coverd by the pre-requirements?

Reply:
You're right. I removed the relevant part from the Pre-requirements. I wanted to keep the install guide to make this whole thing easier for the users to understand.